prov/shm/src/smr_progress.c

```diff
@@ -319,9 +321,26 @@ static int smr_progress_iov(struct smr_cmd *cmd, struct iovec *iov,
 		goto out;
 	}
 
-	ret = smr_cma_loop(peer_smr->pid, iov, iov_count, cmd->msg.data.iov,
-			   cmd->msg.data.iov_count, 0, cmd->msg.hdr.size,
-			   cmd->msg.hdr.op == ofi_op_read_req);
+	/* TODO: Once cma is migrated to the new shm p2p infrastructure,
+	 * we should be able to call ofi_shm_p2p_copy() with the
+	 * appropriate p2p_type, instead of having a separate function
+	 * call for cma
```

Comment:
Why not convert cma to use the new p2p infrastructure in a patch immediately following the patch that adds p2p?  If I'm following the this, p2p is defined, suggests that it supports cma, but doesn't actually do that.  Anyone jumping into the code at this point will be confused.

If CMA isn't being added, I'd comment out the CMA p2p_type until it is added to make it clear that it's not yet there.